_data/projects.yml

```diff
@@ -281,7 +263,15 @@ publicly release, please send an email to <a href=\"mailto:lind-dev@googlegroups
       image: "img/projects/crashsimulator_diagram.png"
       site:
       status: *stealth
-      description: "CrashSimulator is a tool that replicates “real-world” testing for new and upgraded software without the complications of “real-world” deployment. The program enables software developers to identify vulnerabilities in product designs long before they are packaged and released."
+      description: "CrashSimulator is a tool that replicates “real-world” testing for new and upgraded software without 
```

Comment:
"that replicates “real-world” testing" perhaps change "testing" to something that implies environment or errors or similar.

Reply:
Fixed